indexers/staking-squid/schema.graphql

```diff
@@ -0,0 +1,88 @@
+type Operator @entity {
+  id: ID! @index
+  operatorId: Int! @index
+  signingKey: String! @index
+  operatorOwner: String @index
+  currentDomainId: Int
+  nextDomainId: Int
+  minimumNominatorStake: BigInt
+  nominationTax: Int
+  currentTotalStake: BigInt
+  currentEpochRewards: BigInt
+  currentTotalShares: BigInt
+  nominatorAmount: Int! @index
```

Comment:
What does `nominatorAmount` represent? Total amount nominated to the operator?

Reply:
yeah I don't think we need this field as currentTotalStake should be that value